package.json

```diff
@@ -40,7 +40,8 @@
     "prepublishOnly": "npm run build",
     "test": "jest --coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "storyshot": "jest --coverage --coverageDirectory=coverage/visual-regression --collectCoverageFrom='[\"src/components/**\"]' --testMatch=\"<rootDir>/tests/storyshot.test.ts\""
```

Comment:
I think it's a bit confusing to have a whole `jest` configuration in this file, and then override many of its values with command flags here. I understand why we're doing that: we need to run Jest with two separate configurations. I think we should do the following:

- Look for a best practice for this type of situation. 
- If we can't find a consistent, best practice, we should strip the `jest` configuration down to only what's common between the two runs.

Reply:
I found a `--projects ` config option which allow specifying different jest configs in separate files and calling them in the command. This seems to be the best practice.